observer.h

```diff
@@ -1,8 +1,29 @@
 #ifndef OBSERVER_H
 #define OBSERVER_H
+#include"ind_data.h"
 #include "simulation.h"
 #include "Stopwatch.hpp"
 
+///Calculates the reaction_norm of an individual's network
+template<class Ind>
+std::vector<Ind_Data<Ind>> calculate_reaction_norms(const std::vector<Ind>& inds,
+                                                     const range& cue_range,
+                                                     const int& n_data_points)
+{
+    double step_size = (cue_range.m_end - cue_range.m_start)/n_data_points;
+    std::vector<Ind_Data<Ind>> inds_data(inds.size());
+    for(const auto& ind : inds)
+    {
+        std::vector<std::vector<double>> reac_norm(n_data_points);
+        for(double i = cue_range.m_start; i < cue_range.m_end; i += step_size)
+        {
+            auto ind_net = ind.get_net();
+            reac_norm.push_back(output(ind_net, std::vector<double>{i}));
```

Comment:
might be nice to also save the range (might be already done) and the number of data points so the input for which we've got the output can be easily re-calculated, if not saving the input itself